app/views/api/_collection_number.xml.builder

```diff
@@ -8,12 +8,14 @@ xml.tag!(tag,
   xml_datetime(xml, :created_at, object.created_at)
   xml_datetime(xml, :updated_at, object.updated_at)
   if !detail
-    xml_minimal_object(xml, :user, User, object.user_id)
+    xml_minimal_object(xml, :user, :user, object.user_id)
   else
     xml_detailed_object(xml, :user, object.user)
-    xml.observations(number: object.observations.length) do
-      object.observations.each do |observation|
-        xml_detailed_object(xml, :observation, observation)
+    if object.observation_ids.any?
+      xml.observations(number: object.observation_ids.count) do
+        object.observation_ids.each do |observation_id|
+          xml_minimal_object(xml, :observation, :observation, observation_id)
+        end
```

Comment:
Is `:observation, :observation` redundant?

Reply:
Yeah, I noticed that most calls to xml_minimal_object have these two parameters identical now.  But not *all*.  The first is the field name, the second is the type.  In most cases those are the same, but in some like "owner" type "user" they are not.  Probably could refactor them all to use named params and provide a default field name, but it was late and I've already spent way too much time on this "small bug fix".